app/src/main/java/com/bashkevich/androidfundamentals/model/entity/MovieWithActors.kt

```diff
@@ -0,0 +1,21 @@
+package com.bashkevich.androidfundamentals.model.entity
+
+import androidx.room.Embedded
+import androidx.room.Junction
+import androidx.room.Relation
+
+
+data class MovieWithActors(
+    @Embedded val movie: MovieEntity,
+    @Relation(
+        parentColumn = "movie_id",
+        entityColumn = "actor_id",
+        entity = ActorEntity::class,
+        associateBy = Junction(
```

Comment:
Ah, you already use it, then you don't need to insert the Ref class by hands, but do it with Room automatically.